### PR TITLE
use gap for version spacing

### PR DESCRIPTION
**Pull request by chmelevskij**
Fixes bottom status bar version position

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
Yes, I think we must use always gap instead of margin in all the flex items.

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
